docs/book/src/plugins/plugins.md

```diff
@@ -28,4 +28,14 @@ For further information check the design proposal docs [Extensible CLI and Scaff
 
 [plugins-phase1-design-doc]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-1.md
 [plugins-phase1-design-doc-1.5]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-1-5.md
-[extending-cli]: extending-cli.md
\ No newline at end of file
+[extending-cli]: extending-cli.md
```

Comment:
The link alias ought to be always at the bottom of the page.
Could we ensure that before getting this one merged?